common/beerocks/hostapd/configuration.cpp

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: BSD-2-Clause-Patent
+ *
+ * SPDX-FileCopyrightText: 2016-2020 the prplMesh contributors (see AUTHORS.md)
+ *
+ * This code is subject to the terms of the BSD+Patent license.
+ * See LICENSE file for more details.
+ */
+
+#include "configuration.h"
+
+namespace prplmesh {
+namespace hostapd {
+namespace config {
```

Comment:
I wasn't sure it will end up with only one class, I thought that there might be more things in the namespace, and therefore there is a logic in adding this namespace.
e.g. assume that we have a free function named `reload()` in namespace `prplmesh::hostapd` - one can be confused if it is `config::reload()` or `process::reload()`.
Anyhow, as you mentioned, a farther discussion is needed.
I am leaving the namespace as is at the moment.